osf/management/commands/check_crossref_dois.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import requests
+
+from datetime import timedelta
+
+from framework.celery_tasks import app as celery_app
+from website import settings
+from website import mails
+from django.utils import timezone
+from django.core.management.base import BaseCommand
+
+import django
+django.setup()
+
+from osf.models import Preprint
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def check_crossref_dois(dry_run=True):
```

Comment:
Minor: Why default to `True` here but `False` in main?

Reply:
Yeah, this makes no sense, It should default to true everywhere.